shims/promise.js

```diff
@@ -1,4 +1,21 @@
-const RSVP = window.RSVP;
+/* globals require */
+
+function isNodeLikeEnvironment() {
+  const windowAbsent = typeof window === 'undefined';
+  const requirePresent = typeof require === 'function';
+
+  return windowAbsent && requirePresent;
+}
+
+let RSVP;
+
+if (isNodeLikeEnvironment()) {
+  const localRequire = require;
+
+  RSVP = localRequire('rsvp');
+} else {
+  RSVP = window.RSVP;
+}
```

Comment:
Shouldn't be. Just remove the import and use the global promise. Then this shim can go away 😄